app/scripts/controllers/network/createInfuraClient.js

```diff
@@ -23,6 +25,15 @@ export default function createInfuraClient({ network, projectId }) {
   const infuraProvider = providerFromMiddleware(infuraMiddleware);
   const blockTracker = new PollingBlockTracker({ provider: infuraProvider });
 
+  const filterMiddleware = createFilterMiddleware({
+    infuraProvider,
+    blockTracker,
+  });
+  const subscriptionManager = createSubscriptionManager({
+    infuraProvider,
```

Comment:
The provider parameters have the wrong names here:
```suggestion
    provider: infuraProvider,
    blockTracker,
  });
  const subscriptionManager = createSubscriptionManager({
    provider: infuraProvider,
```

Reply:
Corrected here: [469bbed](https://github.com/MetaMask/metamask-extension/pull/16693/commits/469bbed580be40d466cc76a97d7910f727eea5cb)